lib/kzg.ex

```diff
@@ -4,6 +4,11 @@ defmodule Kzg do
   """
   use Rustler, otp_app: :lambda_ethereum_consensus, crate: "kzg_nif"
 
+  # Max size: BYTES_PER_FIELD_ELEMENT * FIELD_ELEMENTS_PER_BLOB
+  @type blob :: binary
+  @type kzg_commitment :: bytes48
+  @type kzg_proof :: bytes48
+
   @spec blob_to_kzg_commitment(Types.blob()) :: {:ok, Types.kzg_commitment()} | {:error, binary()}
```

Comment:
These typespecs should be updated with the new types:


```suggestion
  @spec blob_to_kzg_commitment(blob()) :: {:ok, commitment()} | {:error, binary()}
```

(same for `Bls` functions)

Reply:
updated all the types except blob()